test/vcr_cassettes/PlaidPaymentInitiationTest_test_all_routes.yml

```diff
@@ -495,4 +495,550 @@ http_interactions:
         }
     http_version: 
   recorded_at: Sat, 07 Dec 2019 01:47:07 GMT
+- request:
```

Comment:
looks like api requests just got appended to the vcr casette here (instead of replacing the existing api requests) -- is that intended behavior? i forget the expectation, its not a leading question.

additionally, i'm assuming we already followed https://github.com/plaid/plaid-ruby/blob/master/CONTRIBUTING.md#updating-vcr-cassettes

Reply:
I was following those instructions to generate these. Can double click into this... I didn't really inspect the output of the file after following the instructions...

Reply:
seems like the file is not present, even though it should be... not sure how or why that would be.. .